From: Hannes Schulz <schulz@ais.uni-bonn.de>
Date: Wed, 27 Mar 2013 17:50:10 +0100
Subject: force 4.6 version compiler on debian

---
 CMakeLists.txt |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e1b31d8..5bf937f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,10 @@
 cmake_minimum_required( VERSION 2.6 FATAL_ERROR )
 
+# DEBIAN specific stuff
+set(CMAKE_C_COMPILER CACHE FILEPATH /usr/bin/gcc-4.6)
+set(CMAKE_CXX_COMPILER CACHE FILEPATH /usr/bin/g++-4.6)
+set(CUDA_NVCC_FLAGS CACHE PATH --compiler-bindir /tmp/cuv_compiler)
+
 #
 # If the user specifies -DCMAKE_BUILD_TYPE on the command line, take their
 # definition # and dump it in the cache along with proper documentation,
